django/forms/utils.py

```diff
@@ -1,18 +1,13 @@
 import json
-import sys
+from collections import UserList
 
 from django.conf import settings
 from django.core.exceptions import ValidationError  # backwards compatibility
-from django.utils import six, timezone
+from django.utils import timezone
 from django.utils.encoding import force_text
 from django.utils.html import escape, format_html, format_html_join, html_safe
 from django.utils.translation import ugettext_lazy as _
 
-try:
```

Comment:
I also made this change in https://github.com/django/django/pull/7912 if you want to merge that first to have a separate of concerns (I could make the misc. six removal there also).